### PR TITLE
gobackup 1.3.0 (new formula)

### PR DESCRIPTION
**Pull request by huacnlee**
GoBackup is a fullstack backup tool design for application servers, to backup your databases, files to cloud storages (Local disk, FTP, SCP, S3, GCS, Aliyun OSS ...).

- https://gobackup.github.io
- https://github.com/gobackup/gobackup

[asciicast](https://asciinema.org/a/543564)

### CODE REVIEW

File: Formula/gobackup.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  sha256 "739e0c8beb1a9ab453722e1edccc70197dbd55a8617bdc5086de1ff6414a7e08"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    revision = build.head? ? version.commit : version
+
+    system "go", "build", "-o", bin/"gobackup", "-ldflags",
+      "-s -w -X main.version=#{revision}"
```
Comment (SMillerDev) [inline Formula/gobackup.rb:14]:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{revision}")
```

File: Formula/gobackup.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  end
+
+  test do
+    assert_match revision.to_s, shell_output("#{bin}/gobackup -v")
```
Comment (SMillerDev) [inline Formula/gobackup.rb:18]:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/gobackup -v")
```

File: Formula/gobackup.rb (hunk 1 above)
Comment (huacnlee) [reply Formula/gobackup.rb]: Thanks

**Review by dawidd6 — DISMISSED** [review]

**Review by chenrui333 — DISMISSED** [review]

File: Formula/gobackup.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  desc "CLI tool for backup your databases, files to cloud storages"
+  homepage "https://gobackup.github.io"
+  url "https://github.com/gobackup/gobackup/archive/refs/tags/v1.3.0.tar.gz"
+  sha256 "739e0c8beb1a9ab453722e1edccc70197dbd55a8617bdc5086de1ff6414a7e08"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    revision = build.head? ? version.commit : version
```
Comment (chenrui333) [inline Formula/gobackup.rb:11]: actually there is no head setup in this commit

File: Formula/gobackup.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
+class Gobackup < Formula
+  desc "CLI tool for backup your databases, files to cloud storages"
+  homepage "https://gobackup.github.io"
+  url "https://github.com/gobackup/gobackup/archive/refs/tags/v1.3.0.tar.gz"
+  sha256 "739e0c8beb1a9ab453722e1edccc70197dbd55a8617bdc5086de1ff6414a7e08"
+  license "MIT"
```
Comment (chenrui333) [inline Formula/gobackup.rb:6]:
```suggestion
  license "MIT"
  head "https://github.com/gobackup/gobackup.git", branch: "main"
```

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @huacnlee!

**Pull request closed without merging**